### PR TITLE
bpo-33894: Clarified the tempdir documentation

### PR DESCRIPTION
**Pull request by srinivasreddy**
https://bugs.python.org/issue33894

### CODE REVIEW

File: Misc/NEWS.d/next/Documentation/2018-06-21-01-29-36.bpo-33894.tsesJ.rst [hunk 1]
```diff
@@ -0,0 +1 @@
+Clarified the `tempdir` documentation.
```
Comment (ericvsmith) [inline Misc/NEWS.d/next/Documentation/2018-06-21-01-29-36.bpo-33894.tsesJ.rst]: I don't think this merits a NEWS entry. Let's delete this file.

**Review by ericvsmith — CHANGES_REQUESTED** [review]
Other than not adding the NEWS entry, this looks good. Thanks!

File: Misc/NEWS.d/next/Documentation/2018-06-21-01-29-36.bpo-33894.tsesJ.rst (hunk 1 above)
Comment (srinivasreddy) [reply Misc/NEWS.d/next/Documentation/2018-06-21-01-29-36.bpo-33894.tsesJ.rst]: Done.

**Comment by srinivasreddy** [comment]
I have made the requested changes; please review again.

**Review by ericvsmith — APPROVED** [review]
Looks good to me. Thanks for the change!

**Pull request merged**